tests/std/tests/P2442R1_views_chunk_death/test.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <cassert>
+#include <ranges>
+#include <span>
+
+#include <range_algorithm_support.hpp>
+#include <test_death.hpp>
+using namespace std;
+using ranges::chunk_view;
+
+using test_range = test::range<input_iterator_tag, const int, test::Sized::yes, test::CanDifference::yes,
+    test::Common::yes, test::CanCompare::yes, test::ProxyRef::no>;
+
+static constexpr int some_ints[] = {0, 1, 2, 3};
+
+void test_view_negative_size_forward_range() {
+    auto r        = span<const int>{some_ints};
+    (void) auto v = chunk_view(r, -1); // chunk_view n_ must be positive

```

Comment:
`(void) auto v = expr;` is invalid.
```suggestion
    auto r                  = span<const int>{some_ints};
    auto v [[maybe_unused]] = chunk_view(r, -1); // chunk_view n_ must be positive
```

Reply:
Done, we usually have the attribute in front so I moved it there